include/nlohmann/detail/output/output_adapters.hpp

```diff
@@ -97,7 +97,8 @@ class output_string_adapter : public output_adapter_protocol<CharType>
   private:
     StringType& str;
 };
-
+/// dump to alternative string type, as defined in basic_json template
```

Comment:
Please remove this comment. It's technically correct, but does not help the user or developer.

Reply:
Ok, fixed.